packages/editor-ui/src/api/workflowHistory.ts

```diff
@@ -0,0 +1,31 @@
+import type { IRestApiContext } from '@/Interface';
+import { makeRestApiRequest } from '@/utils';
+import type { WorkflowHistory, WorkflowVersion } from '@/types/workflowHistory';
+import type { HasAtLeastOneKey } from '@/utils/typeHelpers';
+import { valuesToString } from '@/utils/objectUtils';
+
+const workflowHistoryApiRoot = '/workflow-history';
+
+export const getWorkflowHistory = async (
+	context: IRestApiContext,
+	workflowId: string,
+	queryParams: HasAtLeastOneKey<{ take?: number; skip?: number }>,
+): Promise<WorkflowHistory[]> =>
+	makeRestApiRequest(
+		context,
+		'GET',
+		`${workflowHistoryApiRoot}/workflow/${workflowId}?${new URLSearchParams(
+			valuesToString(queryParams),
+		).toString()}`,
```

Comment:
1. Can we hardcode `/workflow-history` like the others? Do we gain much by parameterizing like this vs. readability and searchability? 
2. Intuitively this looks like a lot of work/abstraction to create query params: apply transform to object, pass to `URLSearchParams`, call `toString()`. Could we have something simpler like this? Or is there a benefit I'm not seeing?

```ts
const qsParams = Object.entries(obj)
  .map(([key, value]) => `${key}=${value}`)
  .join('&');
  
path.concat(`?${qsParams}`)  
```